modules/update-service-create-service-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+[id="update-service-create-service-web-console_{context}"]
+= Creating an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to create an Update Service application by using the OpenShift Update Service Operator.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+* The Openshift Update Service graph-data container image has already been created and pushed to a repository accessible to the OpenShift Update Service.
+* The current release and update target releases have already been mirrored to a locally accessible registry.
+
+.Procedure
+
+To create an OpenShift Update Service application by using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Choose *OpenShift Update Service* from the list of installed Operators.
+
+. Click the *Update Service* tab.
+
+. Click *Create UpdateService*.
+
+. Enter a name in the *Name* field, for example, `service`.
+
+. Enter the local pullspec in the *Graph Data Image* field to the graph-data container image created in xref:modules/update-service-graph-data.adoc#update-service-graph-data_{context}[Creating the OpenShift Update Service graph data container image], for example, `registry.example.com/openshift/graph-data:latest`.
+
+. In the *Releases* field, enter the local registry and repository created to contain the release images in xref:modules/update-service-mirror-release.adoc#update-service-mirror-release_{context}[Mirroring the OpenShift Container Platform image repository], for example, `registry.example.com/ocp4/openshift4-release-images`.
+
+. Enter `2` in the *Replicas* field.
+
+. Click *Create* to create the OpenShift Update Service application.
+
+. Verify the OpenShift Update Service application:
++
+** Ensure that the *Update Service* application is listed with *Status Condition: ReconcileCompleted*. If an additional CA certificate had to be configured to allow access to the registry the *Update Service* application is listed with *Status Condition: ReconcileCompleted, RegistryCACertFound*.
```

Comment:
*Status Condition: ReconcileCompleted* does not reflect the pods status. In a situation that update service application pods are not ready due to ImagePullBackOff, we can also get Status Condition: ReconcileCompleted. So to verify the application, we need to check the pod status in the resources tab under the application details page as well.

Reply:
Modified verification to check for all resource status' to be `Created`